reference/sectionC_interpretation.inc

```diff
@@ -12,8 +12,14 @@ Section C: Interpretation
 Interpretation of imports
 -------------------------
 
-#. Each :code:`import` element present in a :ref:`CellML infoset<specA_cellml_infoset>` (the importing infoset) SHALL define a new and separate instance of the CellML infoset referenced by the :code:`href` attribute (the imported infoset).
-   See :ref:`Units reference<specC_units_reference>` and    :ref:`Component reference<specC_component_reference>` for the specifics    of importing units and components.
+#. Each :code:`import` element present in a :ref:`CellML infoset<specA_cellml_infoset>` (the importing infoset) SHALL define a new and distinct instance of the CellML infoset which is referenced by the :code:`href` attribute (the imported infoset).
+   This imported infoset instance SHALL provide the context in which the child of an import element (a :code:`units` or :code:`component` element) is defined and interpreted.
+
+#. The import of an element SHALL be considered recursively such that items referenced by the :code:`import` element child (a :code:`units` or :code:`component` element) are themselves available to that :code:`import` element child.
+   It is noted, for the avoidance of doubt, that the importing infoset SHALL NOT have access to such recursively imported items, but only to explicitly specified :code:`import` element children.
+
+#. The import of an element SHALL be considered recursively such that items referenced by the :code:`import` element's child (a :code:`units` or :code:`component` element) are themselves included.
```

Comment:
Not sure if this is still needed??

Reply:
How so?

Reply:
The two points seem really similar? not sure if the second is a leftover previous version of the first?

Reply:
Oh yeah. I agree drop 2nd one

Reply:
Done in 6078884